.travis.yml

```diff
@@ -1,45 +1,103 @@
 language: php
 sudo: false
+dist: precise
```

Comment:
we should upgrade this to xenial some time in the future

Reply:
See https://docs.travis-ci.com/user/reference/overview/#Virtualisation-Environment-vs-Operating-System for an overview of the environments that Travis provides by default.

Reply:
ah no xenial yet, then precise is definitely fine :+1: